apps/web/components/Stats.tsx

```diff
@@ -0,0 +1,38 @@
+import { StatsInterface } from "../utils/getStatsAndContributors";
+
+interface StatsProps {
+  stats: StatsInterface;
+}
+
+export default function Stats({ stats }: StatsProps) {
+  return (
+    <section className="py-14">
+      <div className="container grid grid-cols-2 gap-y-9 gap-x-12 text-center sm:grid-cols-4">
+        <p className="space-y-1 xl:space-y-2">
```

Comment:
Paragraphs should probably not contain block-level elements inside. Maybe just use div here.